### DISCUSSION TITLE
Validation of thermal comfort indices

### DISCUSSION THREAD

**Discussion by j3r3m1** [UMEP-dev/UMEP#568] [Ideas]
Here is an article using an interesting experiment that could be used for the validation of URock / SOLWEIG.
[https://doi.org/10.1016/j.scitotenv.2021.149527](https://doi-org.insu.bib.cnrs.fr/10.1016/j.scitotenv.2021.149527)

Might be an interesting dataset to compare with UMEP (the dataset does not seem freely available for the moment though).

**Comment by biglimp** [comment]
Can you add the citation please. I am having issues with the link you send.

**Reply by j3r3m1** [reply]
Chen, T., Pan, H., Lu, M., Hang, J., Lam, C. K. C., Yuan, C., & Pearlmutter, D. (2021). Effects of tree plantings and aspect ratios on pedestrian visual and thermal comfort using scaled outdoor experiments. Science of the total environment, 801, 149527.
